tests/unit/secret/output/test_json_output.py

```diff
@@ -147,6 +148,15 @@ def test_json_output(client, cache, name, input_patch, expected_exit_code):
                 json_flat_results
             )
 
+        # all ignore sha should be in the output
+        assert all(
+            ignore_sha in json_flat_results
+            for result in results.results
+            for ignore_sha in leak_dictionary_by_ignore_sha(
+                result.scan.policy_breaks
+            ).keys()
```

Comment:
nit: you can remove `.keys()` here: the loop iterate on dictionary keys by default.